lint/results.go

```diff
@@ -117,18 +117,23 @@ type ResultContext struct {
 }
 
 func (r Result) TtyPrint() {
+	var Reset = "\033[0m"
+	var Red = "\033[31m"
+	var Green = "\033[32m"
+	var Yellow = "\033[33m"
+	var Orange = "\033[38;5;208m"
 	var sym string
 	switch s := r.Severity; s {
 	case Success:
-		sym = "✔️"
+		sym = Green + "✔️" + Reset
 	case Fixed:
-		sym = "❌ (fixed)"
+		sym = Orange + "🛠️ (fixed)" + Reset
 	case Exclude:
 		sym = "➖"
 	case Warning:
-		sym = "⚠️"
+		sym = Yellow + "⚠️" + Reset
 	case Error:
-		sym = "❌"
+		sym = Red + "x" + Reset
```

Comment:
Nice QOL improvements here. For Error, can we still use the red X emoji, or, at least use an upper case X? :)
```suggestion
		sym = Red + "❌" + Reset
```

Reply:
I can't remember where I saw that a long time ago, was that this special `"x"` should be used as the standard for failures so the color can be changed unlike `"❌"` which is always red.  Thats why I changed it but it was so long ago, I don't even remember where it came from.    But I'm good with reverting it.